Sources/Deprecations+Removals.swift

```diff
@@ -324,22 +327,6 @@ extension CompositeDisposable {
 	public func addDisposable(_ d: Disposable) -> Disposable? { fatalError() }
 }
 
-extension Observer {
```

Comment:
This extension is removed since somehow the compiler cannot resolve `extension Signal.Observer` in this particular file.